<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employeequery</class>
 <widget class="QDialog" name="employeequery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关键字</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>升序</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="downButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>降序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>排序方式</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="orderidButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>员工号</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ordertypeButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>员工类型</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
